chef/cookbooks/ceph/recipes/mon.rb

```diff
@@ -91,7 +91,7 @@
   end
 
   execute "ceph-mon mkfs" do
-    command "ceph-mon --mkfs -i #{node['hostname']} --keyring '#{keyring}'"
+    command "chown ceph:ceph #{keyring} ; ceph-mon --mkfs -i #{node['hostname']} --keyring '#{keyring}' --setuser ceph --setgroup ceph"
```

Comment:
Is the `chown` required only when migrating from a previous release, or all the time? I find this slightly suboptimal, but I don't have a better proposal right now.